mailbox/store/src/test/java/org/apache/james/mailbox/store/StoreMailboxManagerAnnotationTest.java

```diff
@@ -149,4 +163,78 @@ public void getAnnotationsByKeysShouldRetrieveStoreAnnotationsByKey() throws Exc
         assertThat(storeMailboxManager.getAnnotationsByKeys(mailboxPath, session, KEYS)).isEqualTo(ANNOTATIONS);
     }
 
+    @Test
+    public void deleteMailboxShouldDoNothingWithAnnotationWhenDoNotSupportAnnotation() throws Exception {
```

Comment:
what is the point the check that, that complexity should be handle at implementation layer (in backend)